zulipterminal/ui_tools/buttons.py

```diff
@@ -132,6 +133,14 @@ def __init__(self, properties: List[Any],
     def set_muted_streams(self) -> None:
         if self.stream_id in self.controller.model.muted_streams:
```

Comment:
If you're adding `self.model`, you can simplify this too.

Reply:
Done :+1: